astropy/utils/tests/test_progress_bar_func.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+import numpy as np
+
+from ..misc import NumpyRNGContext
+
+
+def func(i):
+    """An identity function that jitters its execution time by a
+    pseudo-random amount.
+
+    FIXME: This function should be defined in test_console.py, but Astropy's
+    `python setup.py test` interacts strangely with Python's `multiprocessing`
```

Comment:
@drdavella , do you have any insights on the `python setup.py test` behavior here? If not, that is fine. I am just wondering. Thanks!